File: L2ScriptMaker/Eula.vb
Public Class Eula

    Private Sub Accept2Button_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles Accept2Button.Click
        Me.Close()
        'form.WelcomeForm
    End Sub

    Private Sub DeclineButton_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles DeclineButton.Click
        End
        'Application.Exit()
    End Sub

    Private Sub Eula_Load(ByVal sender As Object, ByVal e As System.EventArgs) Handles Me.Load
        TextBox1.Select(0, 0)
    End Sub
End Class

File: L2ScriptMaker/Eula.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace L2ScriptMaker
{
	public partial class Eula : Form
	{
		public Eula()
		{
			InitializeComponent();
		}

		private void Accept2Button_Click(object sender, EventArgs e)
		{
			this.Close();
		}

		private void DeclineButton_Click(object sender, EventArgs e)
		{
			Environment.Exit(0);
		}

		private void Eula_Load(object sender, EventArgs e)
		{
			TextBox1.Select(0, 0);
		}
	}
}
